src/mailkit.cs

```diff
@@ -0,0 +1,423 @@
+
+		[NullAllowed, Export ("encodedMessage", ArgumentSemantic.Copy)]
+		MEEncodedOutgoingMessage EncodedMessage { get; }
+
+		[NullAllowed, Export ("signingError", ArgumentSemantic.Copy)]
+		NSError SigningError { get; }
+
+		[NullAllowed, Export ("encryptionError", ArgumentSemantic.Copy)]
+		NSError EncryptionError { get; }
+	}
+
+	[NoWatch, NoTV, NoiOS, NoMacCatalyst, Mac (12,0)]
+	[Advice ("'MEMessageSecurityInformation' is not available in UIKit on macOS.")]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MEMessageSecurityInformation : NSSecureCoding
+	{
+		[Export ("initWithSigners:isEncrypted:signingError:encryptionError:")]
+		IntPtr Constructor (MEMessageSigner[] signers, bool isEncrypted, [NullAllowed] NSError signingError, [NullAllowed] NSError encryptionError);
+
+		[Export ("signers", ArgumentSemantic.Strong)]
+		MEMessageSigner[] Signers { get; }
+
+		[Export ("isEncrypted")]
+		bool IsEncrypted { get; }
+
+		[NullAllowed, Export ("signingError", ArgumentSemantic.Strong)]
+		NSError SigningError { get; }
+
+		[NullAllowed, Export ("encryptionError", ArgumentSemantic.Strong)]
+		NSError EncryptionError { get; }
+	}
+
+	[NoWatch, NoTV, NoiOS, NoMacCatalyst, Mac (12,0)]
+	[Advice ("'MEMessageSigner' is not available in UIKit on macOS.")]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MEMessageSigner : NSSecureCoding
+	{
+		[Export ("initWithEmailAddresses:signatureLabel:context:")]
+		IntPtr Constructor (MEEmailAddress[] emailAddresses, string label, [NullAllowed] NSData context);
+
+		[Export ("emailAddresses", ArgumentSemantic.Copy)]
+		MEEmailAddress[] EmailAddresses { get; }
+
+		[Export ("label")]
+		string Label { get; }
+
+		[Export ("context")]
+		NSData Context { get; }
+	}
+
+	[NoWatch, NoTV, NoiOS, NoMacCatalyst, Mac (12,0)]
+	[Advice ("'MEOutgoingMessageEncodingStatus' is not available in UIKit on macOS.")]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MEOutgoingMessageEncodingStatus : NSSecureCoding
+	{
+		[Export ("initWithCanSign:canEncrypt:securityError:addressesFailingEncryption:")]
+		IntPtr Constructor (bool canSign, bool canEncrypt, [NullAllowed] NSError securityError, MEEmailAddress[] addressesFailingEncryption);
+
+		[Export ("canSign")]
+		bool CanSign { get; }
+
+		[Export ("canEncrypt")]
+		bool CanEncrypt { get; }
+
+		[NullAllowed, Export ("securityError", ArgumentSemantic.Copy)]
+		NSError SecurityError { get; }
+
+		[Export ("addressesFailingEncryption", ArgumentSemantic.Copy)]
+		MEEmailAddress[] AddressesFailingEncryption { get; }
+	}
+
+	[NoWatch, NoTV, NoiOS, NoMacCatalyst, Mac (12,0)]
+	[Advice ("'MEAddressAnnotation' is not available in UIKit on macOS.")]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MEAddressAnnotation : NSSecureCoding
+	{
+		[Static]
+		[Export ("errorWithLocalizedDescription:")]
+		MEAddressAnnotation AnnotateError (string localizedDescription);
```

Comment:
These were originally "Error", "Warning", and "Success" but I tried to start them with verbs. Is there a better choice for this?

Reply:
Why annotate? I'm not native, but annotate to me is:
```
add notes to (a text or diagram) giving explanation or comment.
```
What about GetLocalizedError () etc

Reply:
Okay I think these are better names! I was getting confused looking at the web docs, but this makes sense!